cmd/test_utils/basic.go

```diff
@@ -253,7 +253,7 @@ func GetLogFieldsFromMsgs(txMsgs []sdk.Msg) log.Fields {
 			fields[ikeypref+"rcp_name"] = msg.Name
 			fields[ikeypref+"sender"] = msg.Sender.String()
 		case msgs.MsgExecuteRecipe:
-			fields[ikeypref+"type"] = "MsgCreateRecipe"
+			fields[ikeypref+"type"] = "MsgExecuteRecipe"
 			fields[ikeypref+"rcp_id"] = msg.RecipeID
```

Comment:
We need to add types for MsgDisableRecipe and MsgEnableRecipe too

Reply:
Added